frontend/src/scenes/notebooks/Nodes/utils.tsx

```diff
@@ -146,3 +147,20 @@ export function useSyncedAttributes<T extends CustomNotebookNodeAttributes>(
 
     return [parsedAttrs.current, updateAttributes]
 }
+
+export function isCustomNode(node: Node): boolean {
```

Comment:
Reminder for myself - we can do this nicer with the canvas PR, as it autogathers all the custom components